cmake/external/grpc.cmake

```diff
@@ -45,6 +47,7 @@ ExternalProject_Add(
     #    checkout and clean other dirs under third_party
     # 4. remove .git, and package the directory.
     URL "http://paddlepaddledeps.bj.bcebos.com/grpc-v1.8.x.tar.gz"
+    URL_MD5  ${GRPC_DOWNLOAD_SRC_MD5}
```

Comment:
Can put that md5 string directly in here, so that the url and it's md5 are close to each other, later changes will be noticed.

Reply:
Done. Thanks!